Mariuzzo.DO.DataAnnotations/CedulaAttribute.cs

```diff
@@ -21,7 +24,69 @@ public sealed class CedulaAttribute : ValidationAttribute
         /// <returns><code>true</code> if the string value is a valid Cédula, otherwise <code>false</code>.</returns>
         public override bool IsValid(object value)
         {
-            return _cedulaValidator.IsValid(value);
+            var str = value as String;
+            return String.IsNullOrEmpty(str) || (!IgnoreExceptions && _validExceptions.Contains(str)) || _cedulaValidator.IsValid(value);
         }
+        #region Cédulas that do not fill the criteria of the algorithm
+        private readonly string[] _validExceptions =
```

Comment:
I would make this array a static field to avoid recreating it every time a new instance of the validator is created.